conf/HA/st2.conf.sample

```diff
@@ -20,7 +20,8 @@ logging = /etc/st2/logging.rulesengine.conf
 
 [actionrunner]
 logging = /etc/st2/logging.actionrunner.conf
-virtualenv_opts = --always-copy
+# Leave until EL6 is removed. Causes problems with EL7 and user64
+# virtualenv_opts = --always-copy
```

Comment:
Considering it only affects EL7 and EL8, but Ubuntu 14, 16, 18, EL6 are unaffected I'd propose to just include a warning comment for affected systems and keep `virtualenv_opts` as before.

```suggestion
# The line should be commented and 'always-copy' removed when using EL7 or EL8 as it causes virtualenv issues on pack install
virtualenv_opts = --always-copy
```